abis_mapping/templates/incidental_occurrence_data_v2/templates/instructions.md

```diff
@@ -68,7 +68,7 @@ datatype format, and examples.</ins>
 
 ## APPENDICES
 ### APPENDIX-I: VOCABULARY LIST
-Apart from geodeticDatum, the data validation does not require adherence tho the below vocabularies
+Apart from geodeticDatum, the data validation does not require adherence to the below vocabularies
 for each of the fields indicated as having vocabularies. These vocabularies are provided as a 
```

Comment:
"Apart from geodeticDatum, the fields indicated as having vocabularies are not required to adhere to their corresponding vocabulary." ?